internal/btesting/btesting.go

```diff
@@ -0,0 +1,212 @@
+package btesting
+
+import (
+	"flag"
+	"fmt"
+	"github.com/stretchr/testify/require"
```

Comment:
Please put this import item to a separate group.

Reply:
Done